src/state/queries/profile.ts

```diff
@@ -25,28 +25,30 @@ import {STALE} from '#/state/queries'
 import {track} from '#/lib/analytics/analytics'
 
 export const RQKEY = (did: string) => ['profile', did]
+export const profileBasicKey = (handle: string) => ['profileBasic', handle]
```

Comment:
We can technically use the DID for the query key here, but since the majority of our look ups are going to be using the handle, I think this makes more sense. When we use this as placeholder data for `useProfileQuery()` we can use `getQueriesData()` with a `.find()` to get the profile using the DID.

Reply:
Caching twice instead.